<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Result_Show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  
  background-color: rgb(255, 255, 255);
  color: black; /*Цвет для текста*/
  border: 1px solid gray;
  qproperty-alignment:  'AlignVCenter | AlignRight';

}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(85, 170, 0);
    border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#BEBEBE, stop: 1#D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus_minus">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(85, 170, 0);
    border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#BEBEBE, stop: 1#D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(85, 170, 0);
    border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#BEBEBE, stop: 1#D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_division">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  
  background-color: rgb(85, 255, 127);
  color: black; /*Цвет для текста*/
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#FF7832, stop: 1#FF9739);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiplication">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  
  background-color: rgb(85, 255, 127);
  color: black; /*Цвет для текста*/
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#FF7832, stop: 1#FF9739);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#dadbde, stop: 1#f6f7fa);
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#dadbde, stop: 1#f6f7fa);
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#dadbde, stop: 1#f6f7fa);
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  
  background-color: rgb(85, 255, 127);
  color: black; /*Цвет для текста*/
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#FF7832, stop: 1#FF9739);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#dadbde, stop: 1#f6f7fa);
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#dadbde, stop: 1#f6f7fa);
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#dadbde, stop: 1#f6f7fa);
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  
  background-color: rgb(85, 255, 127);
  color: black; /*Цвет для текста*/
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#FF7832, stop: 1#FF9739);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#dadbde, stop: 1#f6f7fa);
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#dadbde, stop: 1#f6f7fa);
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#dadbde, stop: 1#f6f7fa);
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equals">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  
  background-color: rgb(85, 255, 127);
  color: black; /*Цвет для текста*/
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#FF7832, stop: 1#FF9739);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(85, 170, 0);
    border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#BEBEBE, stop: 1#D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: 1px solid gray;
}
QPushButton :pressed {  /*При нажатии на кнопку меняется градиент*/
	background-color: qlineargradient(x1: 0,  y1: 0, x2: 0, y2: 1,
                                       stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
